syntax = "proto3";      //protobuf的版本

package fixbug;     //代码所在的包
option cc_generic_services = true;

message ResultCode
{
    int32 errcode = 1;
    bytes errmsg = 2;
}

//定义登录请求消息类型
message GetFriendListsRequest {
    uint32 userid = 1;
}

//定义登录响应消息类型
message GetFriendListsResponse {
    ResultCode result = 1;
    repeated bytes friends = 2;
}

//rpc方法 - service
service FriendServiceRpc
{
    rpc GetFriendLists(GetFriendListsRequest) returns(GetFriendListsResponse);
}